Governance/introduction.md

```diff
@@ -10,14 +10,15 @@ If you intend to share your data with colleagues for simple purposes, such as sa
 
 ### **[Publishing climate data](publish/publish-intro.md)**
-Publishing data involves uploading them to an accessible respository, and having a persistent identifier (e.g. a [DOI](https://www.doi.org/)) attached in a similar manner to publishing a research paper. This data can then be used and references by other researchers, increasing the scientific impact your dataresearch-ready data can have. Additionally, many journals now require that data on which a paper is based be published in this manner. In this chapter, the various pathways to data publication are presented, with a focus on recommendations for those at Australian research institutions.
+Publishing data involves uploading them to an accessible respository, and having a persistent identifier (e.g. a [DOI](https://www.doi.org/)) attached in a similar manner to publishing a research paper. This data can then be used and references by other researchers, increasing the scientific impact your dataresearch-ready data can have. Additionally, many journals now require that data on which a paper is based be published in this manner. In this section, the various pathways to data publication are presented, with a focus on recommendations for those at Australian research institutions.
 
-### **[Updating published climate data](update/update-intro.md)**
-As with all scientific outputs, errors and inconsistencies can be found in climate research data, oftentimes after publication. This chapter provides guidance to the updating of published data, along with recommendations for actions that can taken prior to the initial publication (such as good versioning practices) that can make this process much easier down the line, as published data may need to updated many times in its lifespan.
+### **[Managing climate data](manage/manage-intro.md)**
+That you published a dataset or are maintaining one or more dataset replicas, it is important to have a management plan well defined before sharing the data with others. These guidelines cover all different aspects of managing data, focusing on the NCI facility to provide a concrete example. These includes organising and managing a data project, documentation, provenance and enabling data access and discovery. 
+As with all scientific outputs, errors and inconsistencies can be found in climate research data, oftentimes after publication. This section also provides guidance to the updating of published data, along with recommendations for actions that can taken prior to the initial publication (such as good versioning practices) that can make this process much easier down the line, as published data may need to updated many times in its lifespan.

```

Comment:
- replace "That" you published a dataset...  with "when"
- actions that can **be** taken 
-  as published data may need to **be** updated